web/src/components/gallery/Gallery.vue

```diff
@@ -190,6 +183,30 @@
         }
       }
 
+    },
+    watch: {
+
+      searchPhrase () {
+
+        this.$router.replace({ query: { search: this.searchPhrase } }).catch(() => {})
+
+        if(this.searchPhrase === '') {
+
+          this.shownPackages = this.packages
+          return
+
+        }
+
+        axios
+          .get(`https://x.nest.land/api/packages`)
```

Comment:
This will eventually need an api, that fetches results from the db by the search phrase, instead of the front end, in order to avoid chrasing the users browsers. 
> For now, this is fine, until we have more than about 400-500 modules